tools/genenotebook/genenotebook_build.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0"?>
+<tool id="genenotebook_build" name="Build a GeneNoteBook" version="@WRAPPER_VERSION@">
+    <description></description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="aggressive"><![CDATA[
+@START_GNB@
+
+#for genome in $genomes:
+    && genenotebook add genome @CONNECT_INFO@ --name '${genome.name}' '${genome.genome}'
+
+    #for annot in $genome.annots:
+        && genenotebook add annotation @CONNECT_INFO@ --genome-name '${genome.name}' '${annot.annotation}'
+        #if $annot.interproscan:
+            && genenotebook add interproscan @CONNECT_INFO@ --format
+            #if $annot.interproscan.is_of_type('gff3'):
+                gff3
+            #else:
+                tsv
+            #end if
+            '${annot.interproscan}'
+        #end if
+
+        #if $annot.eggnog:
+            && genenotebook add eggnog @CONNECT_INFO@ '${annot.eggnog}'
+        #end if
+    #end for
+#end for
+
+## TODO create genenotebook datatype
+## TODO check user accounts
+## TODO gxit
```

Comment:
Are these still relevant?

Reply:
yes it's probably for the next PR on this tool